docs/vdc/networking-virtual-datacenter.md

```diff
@@ -0,0 +1,397 @@
+---
+title: Azure Virtual Datacenter - A Network Perspective  
+description: Learn how to build your virtual data center in Azure
+author: tracsman
+manager: rossort
+tags: azure-resource-manager
+
+ms.service: virtual-network
+ms.date: 04/3/2018
+ms.author: jonor
+---
+
+# Azure Virtual Datacenter: A Network Perspective
+
+## Overview
+Migrating on-premises applications to Azure, even without any significant changes (an approach known as “lift and shift”), provides organizations the benefits of a secured and cost-efficient infrastructure. However, to make the most of the agility possible with cloud computing, enterprises should evolve their architectures to take advantage of Azure capabilities. Microsoft Azure delivers hyper-scale services and infrastructure, enterprise-grade capabilities and reliability, and many choices for hybrid connectivity. Customers can choose to access these cloud services either via the Internet or with Azure ExpressRoute, which provides private network connectivity. The Microsoft Azure platform allows customers to seamlessly extend their infrastructure into the cloud and build multi-tier architectures. Additionally, Microsoft partners provide enhanced capabilities by offering security services and virtual appliances that are optimized to run in Azure.
+
+This article provides an overview of patterns and designs that can be used to solve the architectural scale, performance, and security concerns many customers face when thinking about moving en masse to the cloud. An overview of how to fit different organizational IT roles into the management and governance of the system is also discussed, with emphasis to security requirements and cost optimization.
+
+## What is a Virtual Data Center?
+In the early days, cloud solutions were designed to host single, relatively isolated, applications, in the public spectrum. This approach worked well for a few years. However, as the benefits of cloud solutions became apparent and multiple large-scale workloads were hosted on the cloud, addressing security, reliability, performance, and cost concerns of deployments in one or more regions became vital throughout the life cycle of the cloud service.
+
+The following cloud deployment diagram illustrates some examples of security gaps (red box) and room for optimization network virtual appliances across workloads (yellow box).
+
+[![0]][0]
+
+The Virtual Data Center (vDC) was born from this necessity for scaling to support enterprise workloads, and the need to deal with the problems introduced when supporting large-scale applications in the public cloud.
```

Comment:
Why do we abbreviate it with a small 'v'?

Reply:
I think this was fiat of the original author. I see that it's not the case in other VDC docs. I'll change it.